client/man/ipa-client-automount.1

```diff
@@ -71,21 +66,11 @@ Restore the automount configuration files.
 
 .SH "FILES"
 .TP
-Files that will be always be configured:
+Files that will be configured:
 
 /etc/nsswitch.conf
```

Comment:
An empty line is needed between the 2 files (otherwise they will be displayed on the same line).

Reply:
Sigh, sorry about that. Both are resolved.